buildenv/jenkins/jobs/infrastructure/Cleanup-Nodes.groovy

```diff
@@ -184,10 +184,12 @@ timeout(time: TIMEOUT_TIME.toInteger(), unit: TIMEOUT_UNITS) {
 
                                     // Cleanup zOS datasets 
                                     if (nodeLabels.contains('sw.os.zos')) {
-                                        listcat = sh(script: "tso listcat | grep '${env.USER}' | grep 'JVM' | cut -d. -f 2-", returnStdout: true).trim()
-                                        listcat.split('\n').each {
-                                            sh "tso delete ${it}"
-                                        }
+                                      def listcat = sh(script: "tso listcat | grep '${env.USER}'| grep 'JVM' | cut -d. -f 2-", returnStdout: true).trim()
+                                         if( !listcat.isEmpty()) {
+                                            listcat.split('\n').each {
+                                               sh "tso delete ${it}"
+                                            }                                       
+                                    } 
```

Comment:
Please:
* indent consistently (four spaces per level)
* format `if` statements consistently (space before, not after `(`)
* use a space both sides of `|` consistently
* remove all trailing whitespace (including line 185)